api_pagopa.yaml

```
@@ -37,11 +37,50 @@ paths:
           description: There was an error in retrieving the user data.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/user_extended":
```

since we refer to [these guidelines](https://opensource.zalando.com/restful-api-guidelines/#129) is it possible to name this endpint with dashes ?

```suggestion
  "/user-extended":
```

---

fix into e437d8b